image_generation/stable_diffusion_1_5/cpp/requirements.txt

```diff
@@ -2,5 +2,5 @@
 torch==2.2.2+cpu
 diffusers==0.27.2
 transformers==4.39.3
-optimum-intel[nncf,openvino]==1.16.0
+optimum-intel[nncf,openvino] @ git+https://github.com/huggingface/optimum-intel.git@main
```

Comment:
I think we can try https://github.com/huggingface/optimum-intel/pull/697 and merge if it helps

Reply:
you can proceed with current PR, because optimum is updated here https://github.com/openvinotoolkit/openvino.genai/pull/427

Now, `convert_tokenizers` is not required